Makefile

```diff
@@ -0,0 +1,59 @@
+##@ Makefile
+
+##@ Define the default version
+VERSION ?= latest
+
+tools.airbyte-ci.install: ## Install airbyte-ci
+	##@ Determine the operating system and download the binary
+	@OS=$$(uname); \
+	if [ "$$OS" = "Linux" ]; then \
+		URL="https://storage.googleapis.com/prod-airbyte-cloud-connector-metadata-service/airbyte-ci/releases/ubuntu/${VERSION}/airbyte-ci"; \
+		echo "Linux based system detected. Downloading from $$URL"; \
+	elif [ "$$OS" = "Darwin" ]; then \
+		URL="https://storage.googleapis.com/prod-airbyte-cloud-connector-metadata-service/airbyte-ci/releases/macos/${VERSION}/airbyte-ci"; \
+		echo "macOS based system detected. Downloading from $$URL"; \
+	else \
+		echo "Unsupported operating system"; \
+		exit 1; \
+	fi; \
+	curl -L -f $$URL -o ~/.local/bin/airbyte-ci; \
+
+	##@ Create the directory if it does not exist and make the binary executable
+	mkdir -p ~/.local/bin && chmod +x ~/.local/bin/airbyte-ci; \
+	echo "╔─────────────────────────────────────────────────────────────────────────────────────────────────╗"; \
+	echo "│                                                                                                 │"; \
+	echo "│                                                                                                 │"; \
+	echo "│    /$$$$$$  /$$$$$$ /$$$$$$$  /$$$$$$$  /$$     /$$ /$$$$$$$$ /$$$$$$$$       /$$$$$$  /$$$$$$  │"; \
+	echo "│   /$$__  $$|_  $$_/| $$__  $$| $$__  $$|  $$   /$$/|__  $$__/| $$_____/      /$$__  $$|_  $$_/  │"; \
+	echo "│  | $$  \ $$  | $$  | $$  \ $$| $$  \ $$ \  $$ /$$/    | $$   | $$           | $$  \__/  | $$    │"; \
+	echo "│  | $$$$$$$$  | $$  | $$$$$$$/| $$$$$$$   \  $$$$/     | $$   | $$$$$ /$$$$$$| $$        | $$    │"; \
+	echo "│  | $$__  $$  | $$  | $$__  $$| $$__  $$   \  $$/      | $$   | $$__/|______/| $$        | $$    │"; \
+	echo "│  | $$  | $$  | $$  | $$  \ $$| $$  \ $$    | $$       | $$   | $$           | $$    $$  | $$    │"; \
```

Comment:
Currently this echos twice.

<img width="1737" alt="image" src="https://github.com/airbytehq/airbyte/assets/1325608/9301e4a3-9c76-405f-af4d-1b7cd029450c">


The solution is supposed to be `@echo` but I cant seem to get that to work.

Will keep hacking on it tomorrow

Reply:
The ascii art is one problem. But the bigger problem is we also print out the line of code and the execution of that line of code.

This is really confusing for

```
echo "airbyte-ci is correctly installed at $$EXPECTED_PATH"
```

because you may see a message that implies its correctly installed when its not 😢 

But im happy to remove the ascii art if it gets in the way or if people are not fans

Reply:
>  But the bigger problem is we also print out the line of code and the execution of that line of code.

Oh yes! 
I like the ascii art :)